tests/integration/test_remoted/test_rids/test_config.py

```diff
@@ -0,0 +1,78 @@
+"""
+ Copyright (C) 2015-2021, Wazuh Inc.
+ Created by Wazuh, Inc. <info@wazuh.com>.
+ This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+"""
+
+import pytest
+
+from pathlib import Path
+from wazuh_testing.tools.monitors.file_monitor import FileMonitor
+from wazuh_testing.utils.configuration import get_test_cases_data, load_configuration_template
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.utils.services import control_service
+
+from . import CONFIGS_PATH, TEST_CASES_PATH
+
+from wazuh_testing.modules.remoted.configuration import REMOTED_DEBUG, REMOTED_WORKER_POOL, REMOTED_VERIFY_MSG_ID
+from wazuh_testing.modules.remoted import patterns
+from wazuh_testing.utils.configuration import set_internal_options_conf
+
+# Set pytest marks.
+pytestmark = [pytest.mark.server, pytest.mark.tier(level=1)]
+
+# Cases metadata and its ids.
+cases_path = Path(TEST_CASES_PATH, 'cases_config.yaml')
+config_path = Path(CONFIGS_PATH, 'config_rids.yaml')
+test_configuration, test_metadata, cases_ids = get_test_cases_data(cases_path)
+test_configuration = load_configuration_template(config_path, test_configuration, test_metadata)
+
+daemons_handler_configuration = {'all_daemons': True}
+
+local_internal_options = {REMOTED_DEBUG: '2'}
+
+# Test function.
+@pytest.mark.parametrize('test_configuration, test_metadata',  zip(test_configuration, test_metadata), ids=cases_ids)
+def test_rids_conf(test_configuration, test_metadata, configure_local_internal_options, truncate_monitored_files,
+                            set_wazuh_configuration):
```

Comment:
The `daemons_handler` is included in the fixtures, it must be added so the manager is started/stopped before and after each test.
```suggestion
                            daemons_handler, set_wazuh_configuration):
```